### DISCUSSION TITLE
Rescaling current values when driving cycle is too aggressive

### DISCUSSION THREAD

**Discussion by ssingh-ipa** [pybamm-team/PyBaMM#3704] [Q&A]
I need to model a 51Ah HP prismatic battery cell and simulate its CCCV cycle with I=17A. The cell chemistry is the same as Chen2020 parameter set. I have updated all the relevant parameters in Chen2020. 

My issue is related to your input in the following issue: https://github.com/pybamm-team/PyBaMM/discussions/3015#discussioncomment-6109177

I have encountered a similar problem. The solver crashes until and unless I rescale my current by a factor of x.  If I use a rescaled current, do I also need to rescale the rest of my parameter list? The results obtained from the rescaled current should then be descaled, is that a right assumption? Can you suggest what would be the right way to go ahead with this?

**Comment by TomTranter** [comment]
Hard to know if you don't say what parameters you have changed. I made an app that lets you change a few things to mix electrodes, resize cells and balance the Li concentrations to get an optimal cyclable capacity. https://cellcombinator.streamlit.app/ Might be helpful.

**Reply by ssingh-ipa** [reply]
I have changed the following parameters: 

- "Cation transference number"
- "Nominal cell capacity [A.h]"
- "Cell cooling surface area [m2]"
- "Cell volume [m3]"
- "Electrode height [m]"
- "Electrode width [m]"
- "Initial concentration in negative electrode [mol.m-3]"
- "Initial concentration in positive electrode [mol.m-3]"
- "Maximum concentration in negative electrode [mol.m-3]"
- "Maximum concentration in positive electrode [mol.m-3]"
- "Negative current collector thickness [m]"
- "Negative electrode active material volume fraction"
- "Negative electrode conductivity [S.m-1]"
- "Negative electrode diffusivity [m2.s-1]"
- "Negative electrode thickness [m]"
- "Negative particle radius [m]"
- "Positive current collector thickness [m]"
- "Positive electrode active material volume fraction"
- "Positive electrode conductivity [S.m-1]"
- "Positive electrode diffusivity [m2.s-1]"
- "Positive electrode thickness [m]"
- "Positive particle radius [m]"
- "Separator thickness [m]"
- 'Number of electrodes connected in parallel to make a cell'

My cell chemistry is NMC622, hence none of the pybamm parameters will be exactly correct. Also, the electrode dimension is quite large, approximately 5m width and 0.081m height, due to which the cellcombinator app cannot be used. 

If I only rescale the current, wouldn't that affect the entire simulation outputs? Is it possible that the idea of rescaling suggested in https://github.com/pybamm-team/PyBaMM/discussions/3015#discussioncomment-6109177 is not feasible?

**Reply by TomTranter** [reply]
Ok that's a lot of parameters. What I would do is start with Chen2020 and just change electrode width and nominal capacity and cell volume by the same factor (x) and then run a discharge at 1C and see what happens. For a 1D model this should effectively be running the same simulation each time independently of x. If your voltage curves all look different then some other parameter also needs to change. And start with no thermal too. Build up complexity slowly so that you know what effect each thing you're changing has

**Reply by TomTranter** [reply]
Number of cells in parallel could be the problem. I'm not too sure what that's doing tbh
